CMakeLists.txt

```diff
@@ -108,23 +104,20 @@ if (VULKAN_HEADERS_INSTALL_DIR)
 endif()
 
 find_package(VulkanHeaders REQUIRED CONFIG QUIET)
-set(VULKAN_HEADER_REGISTRY ${CMAKE_CURRENT_SOURCE_DIR}/external/Vulkan-Headers/registry)
+set(VULKAN_HEADER_REGISTRY ${VULKAN_HEADERS_INSTALL_DIR}/share/vulkan/registry)
```

Comment:
Not needed. VulkanHeaders find_package support provides this for users:
https://github.com/KhronosGroup/Vulkan-Headers/blob/main/BUILD.md

```suggestion
```

Reply:
This is resolved in the branch